rsyslog-suse/SPECS/rsyslog.spec

```diff
@@ -145,7 +145,8 @@ dos2unix doc/*.html
 
 %build
 export CFLAGS="$RPM_OPT_FLAGS -fno-strict-aliasing -W -Wall"
-%if 0%{?suse_version} > 1000 && 0%{?suse_version} < 1030
```

Comment:
< 1030 does not match SELS 10 SP3, this changes the behavior on it (which is IIRC our only supported SLES 10). Is it intended?